README.md

```diff
@@ -84,18 +84,26 @@ Overrides the default hostname to connect to an S3-compatible service. i.e, `htt
 
 * `AWS_S3_FORCE_PATH_STYLE`
 
-To enable path-style addressing(i.e., `http://s3.amazonaws.com/BUCKET/KEY`) when connecting to an S3-compatible service that lack of support for sub-domain style bucket URLs (i.e., `http://BUCKET.s3.amazonaws.com/KEY`). Defaults to `false`.
+To enable path-style addressing (i.e., `http://s3.amazonaws.com/BUCKET/KEY`) when connecting to an S3-compatible service that lack of support for sub-domain style bucket URLs (i.e., `http://BUCKET.s3.amazonaws.com/KEY`). Defaults to `false`.
 
-* `WALG_AZURE_BUFFER_SIZE` (eg. `33554432`)
+* `GCS_CONTEXT_TIMEOUT`
+
+Default: 1 hour.
```

Comment:
@x4m @Tinsane I was not sure what is the best way to describe this variable. If you have any suggestions, I can extend this.

Reply:
Currently, any WAL-G specific variable (unlike, e.g. pgx's PGHOST) can be set with either WALG_, WAL_E prefix, or without.

Reply:
I think we should tend to use WALG_ prefix...

Reply:
`WALG_ GC_CONTEXT_TIMEOUT`, `WALG_GC_NORMALIZE_PREFIX` @x4m?